pode.build.ps1

```diff
@@ -224,10 +225,10 @@ Task Build BuildDeps, {
     Push-Location ./src/Listener
 
     try {
-        Invoke-PodeBuildDotnetBuild -target 'netstandard2.0'
-        Invoke-PodeBuildDotnetBuild -target 'net6.0'
-        Invoke-PodeBuildDotnetBuild -target 'net7.0'
-        Invoke-PodeBuildDotnetBuild -target 'net8.0'
+        Invoke-PodeBuildDotnetBuild -target 'netstandard2.0' -Version $Version
+        Invoke-PodeBuildDotnetBuild -target 'net6.0' -Version $Version
+        Invoke-PodeBuildDotnetBuild -target 'net7.0' -Version $Version
+        Invoke-PodeBuildDotnetBuild -target 'net8.0' -Version $Version
```

Comment:
Do we need to pass Version here, as it is available already in the function we're passing it into from the top-level Version parameter 🤔

Reply:
I was thinking it's more clean.
Personally I like to use always params and local variable when possible.
I can remove them if you want. it's not a big deal

Reply:
Solved in #1261